### ISSUE TITLE
[MediaPlayerElement] Corner Radius not working as expected

### ISSUE THREAD

**Issue by DominicMaas** [microsoft/microsoft-ui-xaml#8264]
### Describe the bug

Unable to clip a media player element to have a corner radius. It sometimes works on my secondary monitor, but not my primary, some aspect ratios sometimes work, others not, larger corner radiuses sometimes work, but not after a restart. Bit of an inconsistent mess. 

I found that sometimes making the width / height smaller it will work, sometimes....

### Steps to reproduce the bug

1. Wrap a MediaPlayerElement within a grid with a corner radius, try different radii, widths / heights, aspect ratios.
2. Observe inconsistent corner radius behaviour. 

### Screenshots

<img width="503" alt="image" src="https://user-images.githubusercontent.com/5589453/223979012-0b53bb5a-311a-4d52-bd4b-e185da6e9c62.png">

### NuGet package version

None

### Windows version

Windows 11 (22H2): Build 22621

### Additional context

WinUI: 1.2.230217.4

https://github.com/DominicMaas/Repro_MediaPlayerElementClipping

```xml
<Grid
        HorizontalAlignment="Stretch"
        VerticalAlignment="Stretch"
        Background="Green">
        <StackPanel Spacing="20" HorizontalAlignment="Center" VerticalAlignment="Top">
            <Border
                Width="200"
                Height="200"
                Background="Red"
                CornerRadius="18">
                <MediaPlayerElement
                    AutoPlay="True"
                    CornerRadius="18"
                    Source="ms-appx:///Assets/Video.mp4"
                    Stretch="UniformToFill" />
            </Border>

            <Border
                Width="400"
                Height="200"
                Background="Red"
                CornerRadius="18">
                <MediaPlayerElement
                    AutoPlay="True"
                    CornerRadius="18"
                    Source="ms-appx:///Assets/Video.mp4"
                    Stretch="UniformToFill" />
            </Border>

            <Grid
                Width="100"
                Height="200"
                Background="Red"
                CornerRadius="18">
                <MediaPlayerElement
                    AutoPlay="True"
                    CornerRadius="18"
                    Source="ms-appx:///Assets/Video.mp4"
                    Stretch="UniformToFill" />
            </Grid>
        </StackPanel>
    </Grid>
```

**Comment by sotanakamura** [comment]
I read the issue you mentioned. In your code, MediaPlayerElement is wrapped by layout controls to set CornerRadius. But we expect the MediaPlayerElement's CornerRadius property works. In addtion, Opacity property too.

**Comment by ranjeshj** [comment]
@DominicMaas there have been some fixes in WASDK 1.4 . Can you please try the 1.4 preview 1 and reactivate if you still hit this issue ? Thanks!
